src/cdi/CDI.cc

```diff
@@ -51,6 +51,7 @@ CDI::~CDI() = default;
 //------------------------------------------------------------------------------------------------//
 
 std::vector<double> CDI::frequencyGroupBoundaries = std::vector<double>();
+bool CDI::extend = false;
```

Comment:
Sigh. Global variables always seem to cause problems.  Can you try prepending this line with `DLL_PUBLIC_cdi`? See `c4/C4_MPI.hh` for an example (search for `DLL_PUBLIC_c4`).